Gym_en.rst

```diff
@@ -0,0 +1,213 @@
+                                    spaces.Box(low=-1, high=1, shape=(1,))))
+
+state space
+------------
+
+The state space of Gym-hybrid is represented by a list of 10 elements, which describes the current state of the agent, including the agent's current coordinates, velocity, the sine and cosine of the orientation angle, the coordinates of the target, the distance between the agent and the target, and the target. Distance-related bool value, current relative steps.
+
+.. code::python
+
+    state = [
+                agent.x,
+                agent.y,
+                agent.speed,
+                np.cos(agent.theta),
+                np.sin(agent.theta),
+                target.x,
+                target.y,
+                distance,
+                0 if distance > target_radius else 1,
+                current_step / max_step
+            ]
+
+Bonus space
+-------------
+The reward of each step is set as the length of the agent's distance from the target after the action is performed in the previous step minus the length of the distance from the target after the current step performs the action, ie \``dist_t-1 - dist_t`` \. The algorithm has a built-in \``penalty``\ to motivate the agent to be faster
+Achieve goals. When the episode ends, if the agent stops in the target area, it will get an additional reward with a value of 1; if the agent goes out of bounds or exceeds the maximum number of steps of the episode, it will not get additional reward. The reward at the current moment is expressed by the formula as follows:
+
+.. code::python
+
+    reward = last_distance - distance - penalty + (1 if goal else 0)
+
+
+Termination condition
+------------
+The termination condition for each episode of the Gym-hybrid environment is any of the following:
+
+- The agent successfully entered the target area
+  
+- agant out of bounds
+  
+- Reach the maximum step of the episode
+  
+
+built-in environment
+------------
+There are two built-in environments, \``"Moving-v0"``\ and \``"Sliding-v0"``\. The former does not consider conservation of inertia, while the latter does (so is more practical). The two environments are consistent in state space, action space, and reward space.
+
+other
+====
+
+Store video
+--------
+
+Some environments have their own rendering plug-ins, but DI-engine does not support the rendering plug-ins that come with the environment, but generates video recordings by saving the logs during training. For details, please refer to the Visualization & Logging section under the DI-engine `official documentation <https://opendilab.github.io/DI-engine/quick_start/index.html>`__ Quick start chapter.
+
+DI-zoo runnable code example
+======================
+
+The following provides a complete gym hybrid environment config, using DDPG as the baseline algorithm. Please run the \``gym_hybrid_ddpg_config.py``\ file in the \``DI-engine/dizoo/gym_hybrid``\ directory, as follows.
```

Comment:
The following provides a complete gym hybrid environment config, using DDPG as the baseline algorithm. Please run the \ ``gym_hybrid_ddpg_config.py`` \ file in the \ ``DI-engine/dizoo/gym_hybrid`` \ directory, as follows.